### PR TITLE
fix: set link icon size explicitly

### PR DESCRIPTION
**Pull request by hamza221**
| b | a |
|--------|--------|
| image | image |

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
I think I saw a css var for icon size the other day. We could use that to be even more flexible

**Comment by hamza221** [comment]
> I think I saw a css var for icon size the other day. We could use that to be even more flexible

Couldn't find it, only found ck editor's vars

File: src/components/TextEditor.vue [hunk 1]
```diff
@@ -398,6 +398,9 @@ https://github.com/ckeditor/ckeditor5/issues/1142
  .link-title{
 	color: var(--color-main-text) !important;
  }
```
Comment (GVodyanov) [inline src/components/TextEditor.vue:400]:
```suggestion
 .link-title{
	color: var(--color-main-text) !important;
	margin-left: var(--default-grid-baseline);
 }
```

Comment (GVodyanov) [reply src/components/TextEditor.vue:400]: This looks a little nicer

**Review by GVodyanov — APPROVED** [review]

**Pull request merged**